### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by RealRancor**
Shameless copied from https://github.com/owncloud/calendar/pull/839

Ref: https://github.com/owncloud/core/issues/14609

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* This repository ([notes](https://github.com/owncloud/notes/issues)) is *only* for issues within the ownCloud notes code.
+* __SECURITY__: Report any potential security bug to security@owncloud.com following our [security policy](https://owncloud.org/security/) instead of filing an issue in our bug tracker
+* Report the issue using our [template][template], it includes all the information we need to track down the issue.
+
+Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues.
+
+[template]: https://raw.github.com/owncloud/core/master/issue_template.md
+[mailinglist]: https://mailman.owncloud.org/mailman/listinfo/owncloud
+[forum]: https://forum.owncloud.org/
+[irc]: https://webchat.freenode.net/?channels=owncloud&uio=d4
```
Comment (Raydiation) [inline CONTRIBUTING.md]: should be #owncloud-dev

Comment (RealRancor) [inline CONTRIBUTING.md]: Really? The text is stating "If you have questions about how to install or use ownCloud," so you won't flood #owncloud-dev with installation or usage questions? :-)

Comment (Raydiation) [inline CONTRIBUTING.md]: I thought this was about developer contributions :) Maybe add both then?

**Comment by RealRancor** [comment]
@Raydiation Done

**Pull request merged**